test/unit/deadline_job_attachments/test_download.py

```diff
@@ -2430,7 +2430,7 @@ def download_file(*args):
 
     with patch(
         f"{deadline.__package__}.job_attachments.download.download_file", side_effect=download_file
-    ):
+    ), patch(f"{deadline.__package__}.job_attachments.download.get_s3_client"):
```

Comment:
I noticed that this test was blowing up if the test environment doesn't have valid AWS credentials. It was missing a mock.